apps/prairielearn/src/tests/access.test.ts

```diff
@@ -33,12 +33,12 @@ describe('Access control', { timeout: 20000 }, function () {
 
       Times are:
 
-      1750 before course instance
-      1800 start course instance
-      1850 before assessment
-      1900 start assessment
-      1950 before reservation
-      2000 start reservation
+      1890 before course instance
+      1900 start course instance
+      1910 before assessment
+      1920 start assessment
+      1930 before reservation
+      1940 start reservation
```

Comment:
Man, it was not immediately obvious to me that these were years 🙃 these are some weird tests.

Reply:
Agreed...